scenarios/kube/service_hijacking.yaml

```diff
@@ -0,0 +1,59 @@
+# This scenario allows the user to simulate custom HTTP replies for a kubernetes Service.
```

Comment:
Lets make a readme doc for this with most of these details here

Reply:
Example: https://github.com/krkn-chaos/krkn/blob/main/docs/zone_outage.md

Reply:
:+1: